charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -35,5 +35,23 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: {{ template "zookeeper-operator.fullname" . }}
+        {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+        {{- end }}
+      {{- if default "" (coalesce .Values.imagePullSecrets .Values.global.imagePullSecrets) }}
+      imagePullSecrets:
```

Comment:
The imagePullSercret doesnt need to be added here. It has been configured into the service account.